Objects/Objects/GIS/GisTopography.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using Objects.BuiltElements.Revit;
+using Objects.Geometry;
+using Speckle.Core.Models;
+
+namespace Objects.GIS
+{
+  public class GisTopography : Base
+  {
+    [DetachProperty]
+    public int band_count { get; set; }
+    public List<string> band_names { get; set; }
+    public float x_origin { get; set; }
+    public float y_origin { get; set; }
+    public int x_size { get; set; }
+    public int y_size { get; set; }
+    public float x_resolution { get; set; }
+    public float y_resolution { get; set; }
+    public List<float> noDataValue { get; set; }
+    public List<Mesh> displayValue { get; set; }
```

Comment:
The usage of detachment here is wrong ;) We should have a look at this carefully and pick which ones are chunkable/detachable